status/statusPage.html

```diff
@@ -0,0 +1,699 @@
+                            document.getElementById(idToTest).style.backgroundColor = "forestgreen";
+                        } else if (parsedJSON[i]['status'] === "Failed" && parsedJSON[i]['kill_reason'] === "timeout") {
+                            document.getElementById(idToTest).style.backgroundColor = "darkorange";
+                        } else if (parsedJSON[i]['status'] === "Failed" && parsedJSON[i]['kill_reason'] !== "timeout") {
+                            document.getElementById(idToTest).style.backgroundColor = "red";
+                        }
+                    }
+                }
+            });
+        }
+        determineColor();
+    </script>
+
+    <link href="https://fonts.googleapis.com/css?family=Righteous|Barlow+Condensed|Open+Sans" rel="stylesheet">
+</head>
+<body>
+<header>
+    <img src="https://avatars2.githubusercontent.com/u/32246645?s=280&v=4" alt="DArchR">
+    <h1>UC Davis Computer Architecture Research Group <i>Status Page</i></h1>
+</header>
+<main>
+    <!--- If you want to make a new test, create a div with an ID equal to: "linux-type test-type mem-sys cpu-number boot-type
+    You might notice there area lot of empty div tags, this is to make the grid objects work -->
+    <h2>Mem Sys: classic, Boot Type: init</h2>
+    <div class=box-of-test-grids>
+        <div class="color-code"><span style="color: red;">Failure </span><span
+                style="color: darkorange;">Timeout </span><span style="color: forestgreen;">Success</span></div>
+        <div class="test-grid">
+            <div></div>
+            <div class="test-grid-title">linux-4.4.186</div>
+            <div class="cpu-number-label">8 CPU -</div>
+            <div id="vmlinux-4.4.186 kvm classic 8 init"></div>
+            <div id="vmlinux-4.4.186 atomic classic 8 init"></div>
+            <div id="vmlinux-4.4.186 simple classic 8 init"></div>
+            <div id="vmlinux-4.4.186 o3 classic 8 init"></div>
+            <div class="cpu-number-label">4 CPU -</div>
+            <div id="vmlinux-4.4.186 kvm classic 4 init"></div>
+            <div id="vmlinux-4.4.186 atomic classic 4 init"></div>
+            <div id="vmlinux-4.4.186 simple classic 4 init"></div>
+            <div id="vmlinux-4.4.186 o3 classic 4 init"></div>
+            <div class="cpu-number-label">2 CPU -</div>
+            <div id="vmlinux-4.4.186 kvm classic 2 init"></div>
+            <div id="vmlinux-4.4.186 atomic classic 2 init"></div>
+            <div id="vmlinux-4.4.186 simple classic 2 init"></div>
+            <div id="vmlinux-4.4.186 o3 classic 2 init"></div>
+            <div class="cpu-number-label">1 CPU -</div>
+            <div id="vmlinux-4.4.186 kvm classic 1 init"></div>
+            <div id="vmlinux-4.4.186 atomic classic 1 init"></div>
+            <div id="vmlinux-4.4.186 simple classic 1 init"></div>
+            <div id="vmlinux-4.4.186 o3 classic 1 init"></div>
+            <div></div>
+            <div class=test-type-label>-kvm</div>
```

Comment:
Similarly, this is the x-axis label, correct? I think a better class name would be "x-axis-label"